### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Justman100**
- Strings deleted which are not translated
- Formatted code
- Removed unnecessary notes in the source code

### CODE REVIEW

**Comment by Justman100** [comment]
@SeaDve

File: src/about.rs [hunk 1]
```diff
@@ -12,12 +12,11 @@ pub fn present_window(transient_for: Option<&impl IsA<gtk::Window>>) {
...
         .application_icon(APP_ID)
-        .application_name(gettext("Mousai"))
-        .developer_name(gettext("Dave Patrick Caberto"))
+        .application_name("Mousai")
+        .developer_name("Dave Patrick Caberto")
         .version(VERSION)
-        .copyright(gettext("© 2023 Dave Patrick Caberto"))
+        .copyright("© 2023 Dave Patrick Caberto")
         .license_type(gtk::License::Gpl30)
-        // Translators: Replace "translator-credits" with your names. Put a comma between.
```
Comment (SeaDve) [inline src/about.rs:20]: I'm not sure why is this removed.

File: src/about.rs [hunk 2]
```diff
@@ -72,36 +71,40 @@ fn debug_info() -> String {
     let libpulse_version = pulse::version::get_library_version().to_string_lossy();
 
     format!(
-        r#"- {APP_ID} {VERSION}
```
Comment (SeaDve) [inline src/about.rs:75]: This formatting would be problematic once displayed on the about window.

File: src/core/date_time.rs [hunk 3]
```diff
@@ -23,10 +23,8 @@ impl DateTime {
         let now = Self::now_local();
 
         if self.0.ymd() == now.0.ymd() {
-            // Translators: `%R` will be replaced with 24-hour formatted datetime (e.g., `13:21`)
```
Comment (SeaDve) [inline src/core/date_time.rs:26]: Same with this and another translator comments.

File: src/about.rs (hunk 1 above)
Comment (Justman100) [reply src/about.rs:20]: Like i said! Is superfluous, as these comments are already in the Weblate instance with the respective strings

Comment (SeaDve) [reply src/about.rs:20]: > these comments are already in the Weblate instance with the respective strings

Not sure if I missed some Weblate feature, but it is only displayed from that comment.

image

Comment (sabriunal) [reply src/about.rs:20]: Please keep the comment of traslator-credits. They are visible on translation tools and applications.

resim
